src/main/java/models/Bookstore.java

```diff
@@ -41,7 +41,7 @@ public Bookstore(String name) {
     public void setBookstoreOwner(BookstoreOwner bookstoreOwner) { this.bookstoreOwner = bookstoreOwner; }
     public void removeBookstoreOwner() { this.bookstoreOwner = null; }
 
-    @ManyToMany( cascade=ALL, mappedBy = "bookstores")
+    @ManyToMany( cascade=ALL)
```

Comment:
this is no longer needed?

Reply:
Yes; mappedBy should be in sales so that bookstore handles its sales, not the other way around